client/config/crd/groupsnapshot.storage.openshift.io_volumegroupsnapshotclasses.yaml

```diff
@@ -3,11 +3,10 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/1150"
```

Comment:
@black-dragon74 we should not be removing this one, can you please check why it got removed and add it back, same is applicable to other places